server/controllers/availability.js

```diff
@@ -0,0 +1,37 @@
+const asyncHandler = require('express-async-handler');
+const { google } = require('googleapis');
+const User = require('../models/User');
+
+
+exports.checkAvailablitiy =  asyncHandler(async(res, req, next) => {
+  const { hours } = await User.findOne({ availability: req.params.hours });
```

Comment:
I don't think this is the best way to fetch the User, couldn't there be other users with the same availability? I would do a two part filter where we filter by the owner of the event type for the booking, and the availability.